src/Application.php

```diff
@@ -292,6 +293,11 @@ protected function loadConfig()
             define('CRAFT_TRANSLATIONS_PATH', rtrim($config['craft_translations_path'], '/').'/');
         }
 
+        if (isset($config['dotenv_path'])) {
```

Comment:
Also, can you move the Dotenv loading above this line, before the `$environment` variable is set: https://github.com/tomdavies/craft-cli/blob/4ad9653e4b0b8b6deb5c1ebdd2cad2fa01c7720e/src/Application.php#L270